Source/KerbalConstructionTime/GUI/GUI_Simulation.cs

```diff
@@ -7,7 +7,7 @@ public static partial class KCT_GUI
     {
         private static Rect _simulationWindowPosition = new Rect((Screen.width - 250) / 2, (Screen.height - 250) / 2, 250, 1);
         private static Rect _simulationConfigPosition = new Rect((Screen.width / 2) - 150, (Screen.height / 4), 300, 1);
-        private static Vector2 _bodyChooserScrollPos;
+        private static Vector2 _bodyChooserScrollPos, VABMergeScroll, SPHMergeScroll;
```

Comment:
This is not the correct place for those fields since they have nothing to do with simulations

Reply:
Field names do not follow the same naming convention as others